browser/brave_wallet/solana_provider_impl_unittest.cc

```diff
@@ -688,21 +677,23 @@ TEST_F(SolanaProviderImplUnitTest, Disconnect) {
   ASSERT_TRUE(!account.empty());
   ASSERT_TRUE(IsConnected());
 
+  EXPECT_CALL(*observer_, DisconnectEvent()).Times(1);
```

Comment:
afaik `Times(1)` is a noop
or is it just to be explicit and match vs `Times(0)` ?

Reply:
It was just to explicitly specify one time but I think I can remove it because without WillOnce() nor WillRepeatedly() it would infer just 1

Reply:
fixed and squashed